src/core/server/http/router/request.ts

```diff
@@ -86,11 +86,15 @@ export class KibanaRequest<Params = unknown, Query = unknown, Body = unknown> {
     }
 
     const params =
-      routeSchemas.params === undefined ? {} : routeSchemas.params.validate(req.params);
+      routeSchemas.params === undefined
+        ? {}
+        : routeSchemas.params.validate(req.params, {}, 'params');
```

Comment:
__note:__ :+1: it seems we can even provide some core-based context for the query/params/body schemas at some point, no use case in mind, but it's cool nevertheless :)

Btw, `params`, `query` and `body` may be a little bit ambiguous (e.g. `error: '[query.query]: expected value of type [number] but got [string]'` for query schema with `query` field), what about `RequestParams/Query/Body` or use some separator `(params/query/body)` or something similar just to separate them from the schema part of the context? But it's not a big deal, up to you.